config/content/sass.json

```diff
@@ -1,13 +0,0 @@
-{
```

Comment:
The sass sharing stuff went a while ago but this file was still hanging around. Straggler.